packages/mobile-top-nav/src/assets/img/icon_audio.js

```diff
@@ -0,0 +1,29 @@
+import { html } from 'lit-element';
+import Icon from './icon';
+
+class IconAudio extends Icon {
```

Comment:
Why are you creating a lit-element for each of these instead of just importing the svg string into `ia-icon`?

Reply:
Because when I had them return SVG strings with the fill color interpolated in, you suggested we instead render a component and simply toggle the state rather than redraw the entire SVG object when state changes.

Reply:
I was talking about setting the fill color with CSS instead of interpolating it or is that not possible? This would allow you to add a class to the SVG (or a wrapper div) and then modify the style based on the class.

Reply:
How would we go about changing a fill color for one icon if we're using a CSS variable? The media menu, for example, iterates over the menu options and renders a new instance of `<mediamenu-image>` with the icon type and fill color. That fill color is based on whether the current option is selected. If we render a static SVG string for the icon and have a static CSS var for fill color, we can't update the fill color on a per-icon basis.

Reply:
I'll come up with an approach to this by moving the selected state out a level and toggling fill accordingly. I think this is going to require some CSS duplication, though, as each SVG icon has a different element tree and uses a different combination of fill and stroke attributes on elements. We'll be moving the responsibility of assigning which elements need coloring to the parent element rather than the individual icon.

Reply:
@jbuckner Please check out the static icon approach in src/assets/img/static_icons.js and its usage in src/media-menu.js.

Reply:
We still need the option to change the icon descriptions, too. Componentizing may not be too bad.

Reply:
So are we going the component or plain SVG route?

Reply:
hold at the component since it's already done.